tests/test_cli.py

```diff
@@ -127,6 +127,36 @@ def test_extract_command_with_gelf_parser(gelf_logger):
         assert '{"username": "foo"}' in result.output
 
 
+def test_validate_command_usage():
+    """Test ralph validate command usage."""
+
+    runner = CliRunner()
+    result = runner.invoke(cli, ["validate", "--help"])
+
+    assert result.exit_code == 0
+    assert (
+        "Options:\n"
+        "  -f, --format [edx]     Input format used to validate events  [required]\n"
+        "  -I, --ignore-errors    Continue validating regardless of raised errors\n"
+        "  -F, --fail-on-unknown  Stop validating at first unknown event\n"
+    ) in result.output
+
+    result = runner.invoke(cli, ["validate"])
+    assert result.exit_code > 0
+    assert (
+        "Error: Missing option '-f' / '--format'.  Choose from:\n\tedx."
+    ) in result.output
+
+
+def test_validate_command_with_edx_format():
+    """Test the validate command using the edx format."""
+
+    event = json.dumps(json.loads(PageCloseBrowserEventFactory().json()))
```

Comment:
```suggestion
    event_str = json.dumps(json.loads(PageCloseBrowserEventFactory().json()))
```

Reply:
Nice catch) Thanks) I also removed the `json.dumps(json.loads(` part)

Reply:
Yes. That's a shame I didn't catch it :sweat_smile: